### PR TITLE
Update backlog and appraisal tabs

### PR DESCRIPTION
**Pull request by sallain**
This started out as a pretty simple change to the Backlog tab page, just intending to update some information... and then it spiralled out. Sorry about that!

I've substantially updated the information on both pages to be clearer and more accurate. I've also replaced screenshots throughout both backlog.rst and appraisal.rst and updated formatting to match the style guide where necessary.

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -113,8 +113,7 @@ backlog and allows users to search, download and delete transfers from the
 backlog location.
 
 * :ref:`General description <backlog>`
-* :ref:`Searching for transfers <backlog-transfer-search>`
-* :ref:`Searching for files <backlog-file-search>`
+* :ref:`Searching for transfers and files <backlog-transfer-search>`
 * :ref:`Downloading transfers or files <backlog-download>`
 * :ref:`Deletion of transfers <backlog-delete>`
 * :ref:`Manage a backlog <manage-backlog>`
```
Comment (ablwr) [inline index.rst]: I think this page no longer exists as part of the backlog section and should be removed? I'm not sure if users of the docs actually see this page, but still.

Comment (ablwr) [reply index.rst]: I see this removed in the user-manual.rst though!

**Review by ablwr — CHANGES_REQUESTED** [review]

**Comment by ablwr** [comment]
Only one extremely minor change. Beautiful, beautiful, beautiful docs work!!! :sob: :crown:

File: index.rst (hunk 1 above)
Comment (sallain) [reply index.rst]: Good catch, thanks! I'm surprised that Sphinx didn't catch this.

**Review by ablwr — APPROVED** [review]

File: user-manual/appraisal/appraisal.rst [hunk 2]
```diff
@@ -20,88 +23,148 @@ found in previous versions of Archivematica, also has moved to the Appraisal tab
...
-.. figure:: images/analysis_visualization.*
+.. figure:: images/analysis-objects-visualizations.*
    :align: center
-   :figwidth: 60%
-   :width: 100%
-   :alt: Visualization showing formats by total size of files
+   :width: 80%
+   :alt: The left-hand side of the image shows the backlog pane with a number of files selected. The right-hand side of the image shows a pie chart depicting the number of files for each format.
 
    Visualization showing formats by total size of the selected files
```
Comment (sarah-mason) [inline user-manual/appraisal/appraisal.rst]:
```suggestion
   Visualization showing total number of each format for the selected files
```

**Review by sarah-mason — COMMENTED** [review]
It looks great! Loving the updated photos.

Just one small suggestion for the appraisal section (I hope that I've done it right).

**Comment by sallain** [comment]
@sarah-mason Great catch! Thank you!

**Review by sarah-mason — APPROVED** [review]
I was so excited to find something I read it like 5 times! Really excellent work, it's looking great and easy to follow!

**Pull request merged**